tests/functional/all_daemons/hosts

```diff
@@ -0,0 +1,33 @@
+[mons]
+mon0 monitor_address=192.168.1.10
+mon1 monitor_interface=eth1
+mon2 monitor_address=192.168.1.12
+
+[osds]
+osd0 osd_crush_location="{ 'root': 'HDD', 'rack': 'mon-rackkkk', 'pod': 'monpod', 'host': 'osd0' }"
+osd1 osd_crush_location="{ 'root': 'default', 'host': 'osd1' }"
+
+[mdss]
+mds0
+
+[rgws]
+rgw0
+
+[clients]
+client0
+client1
+
+#[nfss]
+#nfs0
+
+[rbdmirrors]
+rbd-mirror0
+
+[iscsigws]
+iscsi-gw0 ceph_repository="dev"
+
+[all:vars]
+nfs_ganesha_stable=True
```

Comment:
set this to False?

Reply:
dev_setup.yml takes care of this actually

Reply:
Oh OK. Thanks!